app/helpers/tag_helper.rb

```diff
@@ -7,6 +7,9 @@ module TagHelper
     in_progress: "In progress",
     completed: "Completed",
     active: "Active",
+    incomplete: "Incomplete",
+    activating_soon: "Activating soon",
+    reactivating_soon: "Reactivating soon",
```

Comment:
Do we need both of these? I think activating/reactivating might be talking about the same thing, but perhaps have been mistakenly called different things in different areas of the designs. AFAIK both just mean a scheme/location is currently inactive but has a reactivation date in the future.

Reply:
(there's a few other places where these both pop up in this PR that would be affected if I'm right that we only need one)

Reply:
turns out activating soon is required when a scheme has never been deactivated before